_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Until now, NewPipe had a main, pop-up and background -player. All three had separate code and playing queues. Switching from one player to another meant the old player and its queue closed itself, while a new player and queue was created. That sounds complicated and cumbersome, because it is. Moreover, if accidentally switcing using the player buttons below the thumbnail instead of the ones in the drop down menu inside the player, the stream would start anew instead of carrying on from where you left off. Losing your (possibly carefully built) queue was the bonus.
 
-Well, no more! The buttons in the player drop down menu have been removed and the action buttons below the thumbnail work like they're supposed to, so now you're at no risk whatsoever of losing your playing position or queue. Whew!
+No more! There are no buttons in the player drop-down menu anymore, and the action buttons below the thumbnail now only do the job of being action buttons.
+Everyone keeps their position _in the queue_. The utopia is here.
 
-This was made possible by rewriting the code so that all players run on one common service. When you switch from one player to another, this service remains running, preserving the stream data, and the new player just picks up from where the old one left off. Everything remains the same, including the queue. Ah, serendipity.
+An equally complicated and cumbersome code rewrite lets all views use one common service. Switch from one player to another, and the service is at your service preserving the stream data, changing only the view. If anything your blood-pressure could drop _too low_. Ah, serendipity.
 
 ### Video details page
 
-A more unified experience is not the only feature this change brought you. There is one which is immediately noticeable once you open a video in the new version: the video details page. Earlier, when you opened the video details, you were shown a thumbnail, video description, comments, etc. Tapping the thumbnail would open the player in full-screen, hiding the rest of these elements. Now, when you're in portrait mode, the video plays right in the video details page - just like YouTube! There is even a new 'Autoplay' setting which plays the video automatically when you open video details - also just like YouTube! This setting will be turned on by default for Wi-Fi. Of course, you can turn it off if you prefer the behaviour of the previous versions.
+All is one. If you open the video details page you probably think there is going to be a thumbnail, video description, comments, etc.
+Tapping the thumbnail would open the player in full-screen, hiding the rest of these elements. That is no way to live.
+Imagine this, if in portrait mode, the video plays right in the video details page. Raise your hands and say "the day has come". Believe.
+NewPipe now comes equipped with a setting for 'Autoplay'. It can be enjoyed respinsibly for automatic video play upon opening video details.
```

Comment:
```suggestion
NewPipe now comes equipped with a setting for 'Autoplay'. It can be enjoyed responsibly for automatic video play upon opening video details.
```